tools/run_tests/run_interop_tests.py

```diff
@@ -797,6 +848,14 @@ def aggregate_http2_results(stdout):
     server_jobs[lang] = job
     server_addresses[lang] = ('localhost', job.mapped_port(_DEFAULT_SERVER_PORT))
 
+  if args.http2_badserver_interop:
+    # launch a HTTP2 server emulator that creates edge cases
+    lang = str(http2Interop)
+    spec = server_jobspec(Http2Client(), docker_images.get(lang))
```

Comment:
Made this point above, but to give another example, it is confusing that you are instantiating an object called Http2Client, but it is starting a server process